### PR TITLE
Add expeditor

### PR DESCRIPTION
**Pull request by jtimberman**
Add configuration and an action script for automating cookbook
releases for new habitat versions through expeditor

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+slack:
+  notify_channel: ops
+
+subscriptions:
+  - workload: artifact_published:stable:habitat:*
+    actions:
+      - bash:.expeditor/update_hab_release.sh
+      - built_in:update_changelog
```
Comment (tduffield) [inline .expeditor/config.yml]: So, I'm sorry but for the confusion but `artifact_published` isn't a supported for `artifact_published`. It's only supported for `pull_request_merged`. If you want to update the changelog when the PR is merged, that'd be a `merge_action`.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This cookbook provides resources for working with [Habitat](https://habitat.sh).
 
 ### Habitat
 
-- 0.54.0
+- Habitat version: 0.55.0
```
Comment (tduffield) [inline README.md]: Did you mean to check this in?

File: .expeditor/update_hab_release.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+fi
+
+branch="expeditor/${VERSION}"
+git checkout -b ${branch}
+
+sed -i "s/%r{^hab .*\/}/%r{^hab ${VERSION}\/}/" test/integration/config/default_spec.rb
+sed -i "s/%r{^hab .*\/}/%r{^hab ${VERSION}\/}/" test/integration/package/default_spec.rb
+sed -i "s/HAB_VERSION = '.*'.freeze/HAB_VERSION = '${VERSION}'.freeze/" resources/install.rb
+sed -i "s/^- Habitat version: .*/- Habitat version: ${VERSION}/" README.md
+sed -i "s/^version '.*'/version '${VERSION}'/" metadata.rb
```
Comment (tduffield) [inline .expeditor/update_hab_release.sh]: Can we add a VERSION file that also get's updated with the value? That's what Expeditor is looking for to power things like `update_changelog`

Comment (jtimberman) [reply .expeditor/update_hab_release.sh]: line 22!

**Pull request merged**